search-java/src/main/java/uk/os/elements/search/SearchManager.java

```diff
@@ -190,4 +242,24 @@ public SearchBundle call(SearchBundle searchBundle, List<SearchResult> localMatc
         }
         return dest;
     }
+
+    private static AddressApi provideAddressSearchApi() {
+        Retrofit retrofit = new Retrofit.Builder()
+                // TODO
```

Comment:
ah yes, I was going to change that to read: "https://api.ordnancesurvey.co.uk/places/v1" and put addresses in the actual service definition.

e.g. /addresses/find

I'll check some documentation now...